backend/src/main/java/hu/flowacademy/meteo/controller/DataController.java

```diff
@@ -0,0 +1,66 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.*;
+import hu.flowacademy.meteo.model.enumPackage.Type;
+import hu.flowacademy.meteo.service.MeasurementService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.text.ParseException;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class DataController {
+
+    private final MeasurementService measurementService;
+
+    @GetMapping("latest")
+    public MeasurementDto getLastByStationId(@RequestParam("stationId") Long stationId) {
+        log.debug("Geting id of station: {}", stationId);
+        return measurementService.getLastByStationId(stationId);
+    }
+
+    @GetMapping("stations/{stationId}/air")
+    public List<AirDataDto> findAllAirMeasurementsBy(@RequestParam("start") @DateTimeFormat(pattern = "yyyy-MM-dd") Date startDate
+            , @RequestParam("end") @DateTimeFormat(pattern = "yyyy-MM-dd") Date endDate,
```

Comment:
Ha napon belüli, akár egy órás adatokat nem lehet keresni akkor rendben, de ha kisebb intervallumot is vizualizálnátok, ez értelem szerűn nem lesz jó.

Reply:
Jelen állás szerint frontenden hónap/nap szerint lehet lekérni. óra perc szerint nem. :D nyilván  ez még változhat.